File: parcial/src/main/java/co/unicauca/parcial/dto/PersonDTO.java
package co.unicauca.parcial.dto;

import jakarta.persistence.Column;
import lombok.*;

@Builder
@Getter
@Setter
@NoArgsConstructor
@AllArgsConstructor
public class PersonDTO {
    private Integer idPerson;
    private String identificationNumber;
    private String identificationType;
    private String name;
    private String lastName;
}

File: parcial-front/src/app/model/PersonDTO.ts
export class PersonDTO {
    idPerson: number;
    identificationNumber: string;
    identificationType: string;
    name: string;
    lastName: string;

    constructor(
        idPerson: number,
        identificationNumber: string,
        identificationType: string,
        name: string,
        lastName: string
    ) {
        this.idPerson = idPerson;
        this.identificationNumber = identificationNumber;
        this.identificationType = identificationType;
        this.name = name;
        this.lastName = lastName;
    }
}
